src/libsync/clientsideencryptionjobs.h

```diff
@@ -114,7 +114,12 @@ class OWNCLOUDSYNC_EXPORT SetEncryptionFlagApiJob : public AbstractNetworkJob
 {
     Q_OBJECT
 public:
-    explicit SetEncryptionFlagApiJob(const AccountPtr &account, const QByteArray& fileId, QObject *parent = nullptr);
+    enum FlagAction {
+        Clear = 0,
+        Set = 1
+    };
+
+    explicit SetEncryptionFlagApiJob(const AccountPtr &account, const QByteArray& fileId, FlagAction flagAction = Set, QObject *parent = nullptr);
```

Comment:
Space before & not after

Reply:
@er-vin Ok, this was like that before me though :)

Reply:
Likely yes. :-)